cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -349,6 +344,26 @@ instance Ledger.EraPParams era => ToObject (Conway.ConwayGovPredFailure era) whe
     mconcat [ "kind" .= String "MalformedProposal"
             , "govAction" .= govAction
             ]
+  toObject _ (Conway.ProposalProcedureNetworkIdMismatch rewardAcnt network) =
+    mconcat [ "kind" .= String "ProposalProcedureNetworkIdMismatch"
+            , "rewardAccount" .= toJSON rewardAcnt
+            , "network" .= toJSON network
+            ]
+  toObject _ (Conway.TreasuryWithdrawalsNetworkIdMismatch rewardAcnts network) =
+    mconcat [ "kind" .= String "TreasuryWithdrawalsNetworkIdMismatch"
+            , "rewardAccounts" .= toJSON rewardAcnts
+            , "network" .= toJSON network
```

Comment:
^ ping @lehins

Reply:
`network` is the current networkId.
"incorrect" value is in the offending `rewardsAccounts`.
Same for `ProposalProcedureNetworkIdMismatch`

Reply:
What about renaming it as `exptecedNetwork`?

Reply:
I'd also suggest prefixing it with "expected", except I'd call it: `expectedNetworkId`, since `NetworkId` is the name of the type, but it's minor suggestion.

Reply:
Just above there's also this:
```hs
  toObject _ (Conway.ProposalProcedureNetworkIdMismatch rewardAcnt network) =
    mconcat [ "kind" .= String "ProposalProcedureNetworkIdMismatch"
            , "rewardAccount" .= toJSON rewardAcnt
            , "network" .= toJSON network
            ]
```
Should I also change `network` to `expectedNetworkId` as well?

Reply:
> Should I also change network to expectedNetworkId as well?

That's what I did.